src/feeds/direct-pending.feed.ts

```diff
@@ -39,9 +39,6 @@ export class DirectPendingInboxFeed extends Feed<DirectInboxFeedResponse, Direct
 
   async records(): Promise<DirectThreadEntity[]> {
     const threads = await this.items();
-    return threads.map(thread => {
-      const record = new DirectThreadEntity(this.client);
-      return plainToClassFromExist(record, thread);
-    });
+    return threads.map(thread => this.client.entity.directThread(thread.thread_id));
```

Comment:
This is also a breaking change. The data from each thread is no longer in the DirectThreadEntity objects returned.